### PR TITLE
Add Pay Salary Collectives test

### PR DESCRIPTION
**Pull request by NachoPal**
Adding a missing test that was not migrated to this repo when production runtimes were removed from `polkadot-sdk`: https://github.com/paritytech/polkadot-sdk/blob/polkadot-v1.2.0/cumulus/parachains/integration-tests/emulated/collectives/collectives-polkadot/src/tests/fellowship.rs

### CODE REVIEW

File: integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_salary.rs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+			]
+		);
+	});
+
+	AssetHubPolkadot::execute_with(|| {
+		type RuntimeEvent = <AssetHubPolkadot as Chain>::RuntimeEvent;
+		assert_expected_events!(
+			AssetHubPolkadot,
+			vec![
+						RuntimeEvent::Assets(pallet_assets::Event::Transferred { asset_id: id, from, to, amount }) =>
```
Comment (muharem) [inline integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_salary.rs:66]: looks like something wrong here with formatting

File: integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_salary.rs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+			{ 				asset_id: id == &asset_id,
```
Comment (muharem) [inline integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_salary.rs:67]: I think this way of matching just does not work, I would just remove it for now, to not have an illusion that there are some assets

File: integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_salary.rs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	},
+};
+use parachains_common::AccountId;
+use sp_core::crypto::Ss58Codec;
+
+#[test]
+fn pay_salary() {
+	let asset_id: u32 = 1984;
+	let pay_from: AccountId =
+		<AccountId as Ss58Codec>::from_string("13w7NdvSR1Af8xsQTArDtZmVvjE8XhWNdL4yed3iFHrUNCnS")
```
Comment (muharem) [inline integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_salary.rs:32]: lets derive the account id from pallet salary location

**Comment by joepetrowski** [comment]
@NachoPal can you finish up this PR?

**Comment by joepetrowski** [comment]
@muharem could you review again?

**Review by muharem — APPROVED** [review]

**Review by joepetrowski — APPROVED** [review]

**Pull request merged**